style/index.css

```diff
@@ -0,0 +1,370 @@
+body {
+    font-family: "Montserrat", sans-serif;
+    font-optical-sizing: auto;
+    font-weight: normal;
+    font-style: normal;
+}
+
+.center {
+    width: 1150px;
+}
+
+.header__container {
+    background: radial-gradient(rgb(194, 74, 126), rgb(105, 86, 235));
+    height: 750px;
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    flex-direction: column;
+    color: #fff;
+}
+
+/* Menu */
+.nav__section {
+    display: flex;
+    justify-content: space-between;
+    margin: 10px;
+}
+
+.menu {
+    display: flex;
+    height: 100%;
+    justify-content: center;
+    justify-items: center;
+    flex-direction: row;
+}
+
+.logo__nova img {
```

Comment:
w BEM każdy znacznik ma swoją klasę i to jej używamy do stylowania czyli zamiast `img` powinna być klasa